src/main/java/zeus/command/AddDeadlineCommand.java

```diff
@@ -0,0 +1,63 @@
+package zeus.command;
+
+import java.time.LocalDate;
+
+import zeus.exception.ZeusException;
+import zeus.main.Storage;
+import zeus.main.TaskList;
+import zeus.main.Ui;
+import zeus.task.Deadline;
+
+
+/**
+ * Command that handles adding Deadline.
+ */
+public class AddDeadlineCommand extends Command {
+
+    private String description;
+    private String by;
+    private LocalDate ld;
+
+    /**
+     * Constructor of AddDeadlineCommand class.
+     *
+     * @param description Description of Deadline task
+     * @param by String that represents due date
```

Comment:
```suggestion
     * @param description Description of Deadline task.
     * @param by String that represents due date.
```
Might be better to add a punctuation behind each parameter description